priv/repo/migrations/20160802030642_create_note.exs

```diff
@@ -0,0 +1,12 @@
+defmodule Notox.Repo.Migrations.CreateNote do
+  use Ecto.Migration
+
+  def change do
+    create table(:notes) do
+      add :title, :string
+      add :description, :string
+
+      timestamps
```

Comment:
Also, FWIW, you linked to the wrong `timestamps` (the one from Ecto.Schema - the model, not the migration). This is the one you want: https://hexdocs.pm/ecto/Ecto.Migration.html#timestamps/1

It says that the values cannot be null by default, so I think we're good here.